cdap-ui/app/directives/my-global-navbar/my-global-navbar.html

```diff
@@ -0,0 +1,234 @@
+    <div class="brand-header">
+      <a href="" class="navbar-brand" ng-click="Navbar.toggleSidebar()"
+         ng-mouseenter="Navbar.mouseHoverBrand = true"
+         ng-mouseleave="Navbar.mouseHoverBrand = false">
+        <span class="icon-fist"
+              ng-if="Navbar.activeProduct === 'cdap' || Navbar.mouseHoverBrand"></span>
+        <span class="icon-hydrator"
+              ng-if="Navbar.activeProduct === 'hydrator' && !Navbar.mouseHoverBrand"></span>
+        <span class="icon-tracker"
+              ng-if="Navbar.activeProduct === 'tracker' && !Navbar.mouseHoverBrand"></span>
+      </a>
+
+      <a ui-sref="overview({namespace: $stateParams.namespace})"
+         class="menu-item product-title"
+         ng-if="Navbar.activeProduct === 'cdap'">
+        CDAP
+      </a>
+
+      <a ui-sref="hydrator.list({ namespace: $stateParams.namespace })"
+         class="menu-item product-title"
+         ng-if="Navbar.activeProduct === 'hydrator'">
+        Cask Hydrator
+      </a>
+
+      <a ui-sref="tracker-home({ namespace: $stateParams.namespace })"
+         class="menu-item product-title"
+         ng-if="Navbar.activeProduct === 'tracker'">
+        Cask Tracker
+      </a>
+
+    </div>
+
+    <!-- CDAP -->
+    <ul class="navbar-list" ng-if="Navbar.activeProduct === 'cdap'">
+      <li>
+        <a ui-sref="dashboard.standard.cdap({namespace: $stateParams.namespace})"
+           ng-class="{ 'active': Navbar.highlightTab === 'dashboard' }">
+          Dashboard
+        </a>
+      </li>
+      <li>
+        <a ui-sref="admin.overview"
+           ng-class="{ 'active': Navbar.highlightTab === 'management' }">
+          Management
+        </a>
+      </li>
+    </ul>
+
+    <!-- HYDRATOR -->
+    <ul class="navbar-list" ng-if="Navbar.activeProduct === 'hydrator'">
+      <li>
+        <a ui-sref="hydrator.list({ namespace: $stateParams.namespace })"
+           ng-class="{ 'active': Navbar.highlightTab == 'hydratorList'}">
+          My Pipelines
+        </a>
+      </li>
+      <li>
+        <a ui-sref="hydrator.create({ namespace: $stateParams.namespace })"
+           ng-class="{ 'active': Navbar.highlightTab == 'hydratorStudio'}">
+          Create Pipeline
+        </a>
+      </li>
+    </ul>
+
+    <!-- TRACKER -->
+    <ul class="navbar-list" ng-if="Navbar.activeProduct === 'tracker'">
+      <li>
+        <a ui-sref="tracker-home({ namespace: $stateParams.namespace })"
+           ng-class="{ 'active': Navbar.highlightTab == 'search'}">
+          Search
+        </a>
+      </li>
+      <li>
+        <a href=""
+           ng-class="{ 'active': Navbar.highlightTab == 'integration'}">
+          Integrations
+        </a>
+      </li>
+      <li>
+        <a href=""
+           ng-class="{ 'active': Navbar.highlightTab == 'integration'}">
+          Help
+        </a>
+      </li>
+    </ul>
+
+
+    <!-- RIGHT MENUS -->
+    <div class="pull-right right-menu">
+      <!-- <div class="navbar-item">
```

Comment:
Yup, this will be used when we bring back the global search after design is finalized.